routes/web.php

```diff
@@ -234,7 +236,7 @@
             Route::post('/{year}/{hours}/{id}', 'AdminDash@setPyriteWinner');
             Route::get('/remove/{id}/{year}', 'AdminDash@removePyriteWinner');
         });
-        Route::prefix('feedback')->middleware('permission:snrStaff')->group(function () {
+        Route::prefix('feedback')->middleware('permission:staff')->group(function () {
```

Comment:
Did you clear this by Max or Eli? I wouldn't want to change this unless we talk to them first

Reply:
Hmm this shouldn't have changed. That must've snuck in somehow but I'll revert that.

Reply:
Resolved and squashed into 4ea15ae